test/suites/standard2/validate-custom-certificates.robot

```diff
@@ -37,14 +37,14 @@ Test Expired Cert
     [Setup]    Setup Test
     # Generate CSR Config
     Create Keys
-    Create Cert    TestCN    test-expired.api.com    5.5.5.5    0
+    ${hostname}=    Generate Random HostName
+    Create Cert    TestCN    ${hostname}    5.5.5.5    0
     Upload Certificates
     Configure Named Certificates    ${TMPDIR}/server.crt    ${TMPDIR}/server.key
     Restart MicroShift
-    Add Entry To Hosts    ${USHIFT_HOST}    test-expired.api.com
-    Setup Custom Kubeconfig    test-expired.api.com
+    Add Entry To Hosts    ${USHIFT_HOST}    ${hostname}
```

Comment:
We need to clean this up or these host names will pile up in /etc/hosts on dev machines.
Can we delete this entry in Teardown?

Reply:
done